src/evict/evict_lru.c

```diff
@@ -2323,8 +2331,9 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
     *slotp += (u_int)(evict - start);
     WT_STAT_CONN_INCRV(session, cache_eviction_pages_queued, (u_int)(evict - start));
 
-    __wt_verbose_debug2(session, WT_VERB_EVICTION, "%s walk: seen %" PRIu64 ", queued %" PRIu64,
-      session->dhandle->name, pages_seen, pages_queued);
+    __wt_verbose_debug1(session, WT_VERB_EVICTION,
```

Comment:
This one can generate 100s of lines per second. Is it worth pulling it to debug1?

Reply:
Done.